content/doc/installation/with-docker.en-us.md

```diff
@@ -17,111 +17,179 @@ menu:
-```
-docker run -d --name=gitea -p 10022:22 -p 10080:3000 -v /var/lib/gitea:/data gitea/gitea:latest
-```
-
-Now you should have a running instance of Gitea, to access the web UI just visit http://hostname:10080 in your favorite browser, if you want to clone repositories you can do it in the above case with `git clone ssh://git@hostname:10022/username/repo.git`.
-
-`hostname` should be replaced by ip adress of your gitea server.
-
-## Named Volumes 
-
-This guide will result in both the Gitea and PostgreSQL data stored in Docker named volumes. This makes for easy backup, restore and upgrades.
-
-### The Database
-
-First get the named volume ready:
-
-```
-$ docker volume create --name gitea-db-data
-```
-
-Now that we have a volume ready, we can pull and run the PostgreSQL image. This is also based upon Alpine Linux just like Gitea. This will mount the named volume at the correct location instead of a host dir.
-
-```
-$ docker pull postgres:alpine
-$ docker run -d --name gitea-db \
-    -e POSTGRES_PASSWORD=<PASSWORD> \
-    -v gitea-db-data:/var/lib/postgresql/data \
-    -p 5432:5432 \
-    postgres:alpine
-```
-
-Now the database is up and running, we need to configure it. Make sure you remember the password for when Gitea starts.
-
-```
-$ docker exec -it gitea-db psql -U postgres
-psql (9.6.1)
-Type "help" for help.
-
-postgres=# CREATE USER gitea WITH PASSWORD '<PASSWORD>';
-CREATE ROLE
-postgres=# CREATE DATABASE gitea OWNER gitea;
-CREATE DATABASE
-postgres=# \q
-$
-```
-
-### Gitea
-
-First, the named volume
-
-```
-$ docker volume create --name gitea-data
-```
-
-Now run (remove the dns entry if not needed):
-
-```
-$ docker run -d --name gitea \
-	--link gitea-db:gitea-db \
-	--dns 10.12.10.160 \
-	-p 11180:3000 \
-	-p 8322:22 \
-	-v gitea-data:/data \
-	gitea/gitea:latest
-```
-
-You should now have two Docker containers for Gitea and PostgreSQL plus two Docker named volumes and you should have a running instance of Gitea, to access the web UI just visit http://hostname:11180/install/ in your favorite browser.
-
-In the web UI set:
-- Database Type: PostgreSQL
-- Host: gitea-db:5432
-- User: gitea
-- Password: <PASSWORD>
-- ...
-- HTTP port: 11180
-- Application URL: http://hostname:11180
-
-Apply settings. You should be able to access to web UI now and create your first user.
+This reference setup guides you through the setup based on `docker-compose`, the installation of `docker-compose` is out of scope of this documentation. To install `docker-compose` follow the official [install instructions](https://docs.docker.com/compose/install/).
+
+## Basics
+
+The most simple setup just creates a volume and a network and starts the `gitea/gitea:latest` image as a service. Since there is no database available you can start it only with SQLite3. Create a directory like `gitea` and paste the following content into a file named `docker-compose.yml`.
+
+```yaml
+version: "2"
+
+networks:
+  gitea:
+    external: false
+
+services:
+  server:
```

Comment:
I think `gitea` or `git` would be better name. Only `server` doesn't tell you anything about service and you can have more than gitea in your compose file.

Reply:
If you know docker-compose the service gets prefixed by the folder name, which is in case of the docs `gitea` which will result in a container named `gitea_server` which seems to be totally fine for me.

Reply:
Services gets perfixed by project name. If you don't define it by other way, folder name will be used as project name. Container name in your example will be `gitea_server_1` and only if user creates folder with name `gitea` (you say "Create a directory like ..."). I think it should be clear what service is about if you just look at list of services (user should know what is going only from yaml file, not from combination of directory and file - I think that's not a good practice), also it's better for use with `docker-compose` commands (eg `docker-compose logs -f server` doesn't look nice) as only service name is used here, not container name.

Reply:
This looks totally fine as it IS the server.

Reply:
Server is too general (Home PC, RPi, hardware in rack, Tomcat, Proxy, .... everything can be called server). If you have more projects like this (or servers inside one yaml file), how you know what "server" did you mean?